paddle/optimizer/adadelta_optimizer.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PADDLE_ADADELTA_OPTIMIZER_H_
+#define PADDLE_ADADELTA_OPTIMIZER_H_
+
+#include "parameter_optimizer.h"
+
+namespace paddle {
+namespace optimizer {
+
+template <class T>
+class AdadeltaOptimizer : public ParameterOptimizer<T> {
+public:
+  /*! \brief call the applySGD for example  */
+  void update(const Tensor<T> &gradient) {
```

Comment:
Since this is not a working optimizer, maybe let's remove it for now to avoid confusion? (see: https://github.com/PaddlePaddle/Paddle/pull/2190#discussion_r119491357)